start_cerberus.py

```diff
@@ -111,6 +112,24 @@ def main(cfg):
                              "to True and assuming that the nodes are ready")
                 watch_nodes_status = True
 
+            # Monitor cluster operators status
+            if watch_cluster_operators:
+                operators_status = runcommand.invoke("kubectl get co -o yaml")
```

Comment:
Also, I think it would be nice to move invoke and load command within monitor_cluster_operator function. What are your thoughts @chaitanyaenr ?

Reply:
I can definitely move the load yaml to the monitor_cluster_operator function, but right now the invoke is only used in start_cerberus.py file so I was trying to keep that consistent. I agree that it would make it cleaner though!

Reply:
The invoke command is used in other files as well ( cerberus/cerberus/inspect/inspect.py and cerberus/cerberus/slack/slack_client.py). We could use invoke command in this file as well.

Reply:
Also, Paige you could force push a commit to avoid squashing.

Reply:
+1 for moving this logic into the function so that we can call it anywhere to get the status of the cluster operators.

Reply:
Added this change in newest commit